couchbase/src/main/scala/akka/stream/alpakka/couchbase/CouchbaseDocument.scala

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (C) since 2016 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.couchbase
+
+import scala.reflect.ClassTag
+
+class CouchbaseDocument[T: ClassTag](val id: String, val document: T) {
```

Comment:
Is the constructor part of the public API also for Java? (That could make sense for example for users constructing `CouchBaseDocument` instances in their tests) In that case `T : ClassTag` is problematic and there should be an explicit constructor for Java accepting `Class<T>`.

Also, make it `final` if not intended for user subclassing as public API.